setup.cfg

```diff
@@ -28,7 +28,7 @@ all_files = 1
 universal = 1
 
 [pydocstyle]
-add_ignore = D401
+add_ignore = D202,D401
```

Comment:
This implies we need cleanup of `#noqa` in some places?

Reply:
Good point! I missed one actually. Please re-approve.